astropy/wcs/src/unit_list_proxy.c

```diff
@@ -186,6 +187,35 @@ PyUnitListProxy_getitem(
   return result;
 }
 
+static PyObject*
+PyUnitListProxy_richcmp(
+	PyObject *a,
+	PyObject *b,
+	int op){
+  PyUnitListProxy *lhs, *rhs;
+  assert(a != NULL && b != NULL);
+  if (!PyObject_TypeCheck(a, &PyUnitListProxyType) ||
+      !PyObject_TypeCheck(b, &PyUnitListProxyType)) {
+    Py_RETURN_NOTIMPLEMENTED;
+  }
+  if (op != Py_EQ && op != Py_NE) {
+  	Py_RETURN_NOTIMPLEMENTED;
```

Comment:
inconsistent indentation

Reply:
I don't know why it's happening I always try to keep it correct in text editor but when I update this happen can you suggest anything

Reply:
Depends on the text Editor. What do you use?

Reply:
I am using sublime

Reply:
There should be settings for indentations in sublime, in case you want it file-extension dependent, I also found this on StackOverflow: ["Sublime tab size per file type?"](https://stackoverflow.com/q/28977315/5393381)